manuscript/03-flexibles-design.md

```diff
@@ -104,7 +104,7 @@ Manchmal schneiden Leute die Funktionalität in kleine Teile, um eine flexible K
 
 Wenn Elemente eines Modells oder Entwurfs in ein monolithisches Konstrukt eingebettet sind, wird deren Funktionalität dupliziert. Die externe Schnittstelle sagt nicht alles, was einem Verwender wichtig sein könnte. Ihre Bedeutung ist schwer zu verstehen, da verschiedene Konzepte miteinander vermischt werden.
 
-Umgekehrt kann die Zerlegung von Klassen und Methoden den Client sinnlos verkomplizieren, indem sie Clients dazu zwingt, zu verstehen, wie kleine Teile zusammenpassen. Schlimmer noch, ein Konzept kann völlig verloren gehen. Die Hälfte eines Uranatoms ist nicht Uran. Natürlich kommt es nicht nur auf die Grösse der Elemente an, sondern auch, wie gut diese zusammen funktionieren.
+Umgekehrt kann die Zerlegung von Klassen und Methoden den Client sinnlos verkomplizieren, indem sie Clients dazu zwingt, zu verstehen, wie kleine Teile zusammenpassen. Schlimmer noch, ein Konzept kann völlig verloren gehen. Die Hälfte eines Uranatoms ist nicht Uran. Natürlich kommt es nicht nur auf die Größe der Maserung an, sondern auch, wie gut die Maserung verläuft.
```

Comment:
Ich weiss nicht, ob ich das verstehen würde. Und ich habe den Eindruck, dass es mit Maserung nicht die gleiche Bedeutung hat. Aber vielleicht ist meiner Interpretation der Redewendung auch falsch. Meinen Vorschlag würde ich zumindest verstehen, und er ergibt im Kontext irgendwie auch Sinn.

Reply:
Im Original heißt es "...where the grain runs". Meiner Meinung nach ist das definitiv die Maserung, siehe https://idioms.thefreedictionary.com/run+against+the+grain . Ein Korn läuft nicht. Die metaphorische Bedeutung kann man IMHO nur durch eine Erklärung hinbekommen.

Reply:
Meine Interpretation mit dem Korn war, dass es nicht nur auf die Grösse des einzelnen Korns ankommt, sondern auch darauf, wie das Getriede als Ganzes funktioniert, also z.B. wie es durch einen Trichter läuft - je nach "Dichte" des Getreides und der Grösse des Trichters blockieren sich die Körner gegenseitig und nichts geht durch. Aber wie gesagt, ist meine Interpretation im Kontext. Ich habe mal Unterstützung von einer Native English Person angefragt.

Reply:
Die Redewendung ist meiner englischen Kollegin nicht bekannt. Vielleicht handelt es sich dabei um eine Redewendung aus den USA. Sie würde meiner Interpretation aber folgen ;-)